entries/server-entry.js

```diff
@@ -31,6 +29,10 @@ into `__webpack_require__.p = ...` and uses it for HMR manifest requests
 // eslint-disable-next-line
 __webpack_public_path__ = getCompilationMetaData().webpackPublicPath + '/';
 
+// The shared entry must be imported after setting __webpack_public_path__
+// eslint-disable-next-line import/no-unresolved, import/no-extraneous-dependencies
+const main = require('__FRAMEWORK_SHARED_ENTRY__');
```

Comment:
Can you add a comment as to why this is a require?

Reply:
Sure, will expand on the above comment.